### PR TITLE
Various Updates

### PR DESCRIPTION
**Pull request by marksou**
Updated the Create Users page to clarify password issue. Also, fixes issue #169 
Updated Expiry and Quota settings issue.
Changed Tyk Component diagrams

### CODE REVIEW

File: tyk-docs/content/control-limit-traffic/token-expiry.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Tyk manages timestamps in the Unix timestamp format - this means that when a dat
 
 Key sessions are created and updated using the Tyk REST API, in order to set the expiry date for a key, update the `expires` value with the timestamp of when the key should expire.
 
-Leave this field empty for a token never to expire.
+> **Note:** `expires` can only be a positive number, or -1 (unlimited). Zero (0) or other negative numbers, will not work.
```
Comment (mvdan) [inline tyk-docs/content/control-limit-traffic/token-expiry.md:24]: I'd say "will not work" is incorrect, as it is closer to undefined behavior. I didn't know empty string was documented before, so we should keep it defined as unlimited. So I'd write something like:

> `expires` can only be a positive number, or -1 (unlimited). Empty string will be interpreted as -1.

File: tyk-docs/content/get-started/with-tyk-cloud/tutorials/create-security-policy.md [hunk 2]
```diff
@@ -47,7 +47,10 @@ A rate limit is enforced on all keys, set the number of requests per second that
 ![Quota form][5]
 
 Usage quotas limit the number of total requests a user is allowed to have over a longer period of time. So while a rate limit is a rolling window, a quota is an absolute maximum that a user is allowed to have over a week, a day or a month.
-**NOTE:** The Usage Quota set by a policy will override a quota applied to an individual key.
+
+Usage quotas can only be a positive number, or -1 (unlimited). Zero (0) or other negative numbers, will not work.
```
Comment (mvdan) [inline tyk-docs/content/get-started/with-tyk-cloud/tutorials/create-security-policy.md:51]: Same as above - I'd just delete the last sentence. Or if you want to be explicit, say "undefined behavior".

**Review by mvdan — COMMENTED** [review]
the comments apply to all "will not work" changes.

File: tyk-docs/content/control-limit-traffic/token-expiry.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/control-limit-traffic/token-expiry.md:24]: Changed all as requested

**Pull request merged**